src/api/dotnet/Microsoft.Z3.csproj.in

```diff
@@ -0,0 +1,243 @@
+  <PropertyGroup Condition="'$(Platform)' == 'x86'">
+    <CodeAnalysisUseTypeNameInSuppression>true</CodeAnalysisUseTypeNameInSuppression>
+    <CodeAnalysisModuleSuppressionsFile>GlobalSuppressions.cs</CodeAnalysisModuleSuppressionsFile>
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSetDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\\Rule Sets</CodeAnalysisRuleSetDirectories>
+    <CodeAnalysisRuleDirectories>;C:\Program Files (x86)\Microsoft Visual Studio 10.0\Team Tools\Static Analysis Tools\FxCop\\Rules</CodeAnalysisRuleDirectories>
+  </PropertyGroup>
+
+  <!-- Build properties -->
+  <PropertyGroup>
+    <!-- Add net40;net35 only if the appropriate SDKs are installed. -->
+    <!-- Also, in xplat builds, netfx TFMs are not available. -->
+    <TargetFrameworks>netstandard2.0;net461;net45</TargetFrameworks>
+    <OutputTypeEx>library</OutputTypeEx>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <NoWarn>1701,1702</NoWarn>
+    <Warn>4</Warn>
+    <DefineConstants Condition="'$(TargetFramework)'=='net35'">FRAMEWORK_LT_4</DefineConstants>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <DocumentationFile>$(OutputPath)\Microsoft.Z3.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <!-- Compilation items -->
+  <ItemGroup>
+${Z3_DOTNET_COMPILE_ITEMS}
+  </ItemGroup>
+
+  <!-- Legacy .NET framework native library helper routines  -->
+  <ItemGroup>
+    <Content Include="${CMAKE_CURRENT_LIST_DIR}/Microsoft.Z3.props">
+      <PackagePath>build</PackagePath>
+    </Content>
+    <Content Include="${CMAKE_CURRENT_LIST_DIR}/Microsoft.Z3.targets">
+      <PackagePath>build</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <!-- TODO we may want to pack x64 and x86 native assemblies into a single nupkg -->
+
+  <!-- Native binaries x64 -->
+  <ItemGroup Condition="'$(Platform)' == 'x64'">
+    <Content Include="${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$(Configuration)/libz3.dll" Condition="Exists('${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$(Configuration)/libz3.dll')">
+      <PackagePath>runtimes\win-x64\native</PackagePath>
+    </Content>
+    <Content Include="${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libz3.so" Condition="Exists('${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libz3.so')">
+      <PackagePath>runtimes\linux-x64\native</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <!-- Native binaries for x86, note the platform condition is set to "AnyCPU" -->
+  <ItemGroup Condition="'$(Platform)' == 'AnyCPU'">
+    <Content Include="${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$(Configuration)/libz3.dll" Condition="Exists('${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$(Configuration)/libz3.dll')">
+      <PackagePath>runtimes\win-x86\native</PackagePath>
+    </Content>
+    <Content Include="${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libz3.so" Condition="Exists('${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libz3.so')">
+      <PackagePath>runtimes\linux-x86\native</PackagePath>
```

Comment:
I don't think dotnet core is supported on x86 linux

Reply:
You're right. A copy/paste mistake.

Reply:
updated.